testdata/BUILD

```diff
@@ -212,6 +214,25 @@ container_image(
     mode = "0o644",
 )
 
+# Test new_container_load
+container_image(
+    name = "new_docker_tarball_base",
```

Comment:
new_docker_tarball_base suggests it's using a tarball as base. But new_pause_tar seems pull the image as oci.